graal-js/src/com.oracle.js.parser/src/com/oracle/js/parser/Parser.java

```diff
@@ -3625,10 +3628,23 @@ private Expression primaryExpression(boolean yield, boolean await) {
                 markThis();
                 return new IdentNode(primaryToken, finish, name).setIsThis();
             case IDENT:
+                // mimic code from AbstractParser.getIdent()
+                final long identToken = token;
+                String identString = (String) getValue(identToken);
+
+                if (identString.equals("module") && lookahead().equals(LBRACE)) {
```

Comment:
You can use a `static final String MODULE = "module"` here

Reply:
Made the constant.